### PR TITLE
Make eos-paygd use the watchdog timer

### PR DESCRIPTION
**Pull request by mwleeds**
https://phabricator.endlessm.com/T27051

### CODE REVIEW

**Comment by mwleeds** [comment]
Per my comment on the ticket, don't merge this yet.

**Comment by ManMower** [comment]
LGTM

**Comment by mwleeds** [comment]
I updated a comment to explain why we ping the watchdog even if PAYG is not active.

File: eos-paygd/main.c [hunk 1]
```diff
@@ -154,7 +207,10 @@ main (int   argc,
        * initramfs difficult. */
       timeout_id = g_timeout_add_seconds (10 * 60, sync_and_poweroff, NULL);
       while (access ("/etc/initrd-release", F_OK) >= 0)
-        g_usleep (G_USEC_PER_SEC / 5);
+        {
+          g_main_context_iteration (NULL, TRUE); /* keep pinging watchdog */
+          g_usleep (G_USEC_PER_SEC / 5);
```
Comment (treitter) [inline eos-paygd/main.c]: nitpick: I think it'd be clearer if this were expressed like `200 * USEC_PER_MSEC`

Comment (mwleeds) [reply eos-paygd/main.c]: Honestly to me that seems more esoteric

**Pull request merged**